frontend/public/components/app.jsx

```diff
@@ -80,37 +80,57 @@ const EnhancedProvider = ({ provider: ContextProvider, useValueHook, children })
   return <ContextProvider value={value}>{children}</ContextProvider>;
 };
 
-class App_ extends React.PureComponent {
-  constructor(props) {
-    super(props);
-
-    this._onNavToggle = this._onNavToggle.bind(this);
-    this._onNavSelect = this._onNavSelect.bind(this);
-    this._onNotificationDrawerToggle = this._onNotificationDrawerToggle.bind(this);
-    this._isDesktop = this._isDesktop.bind(this);
-    this._isMobile = this._isMobile.bind(this);
-    this._onResize = this._onResize.bind(this);
-    this.previousDesktopState = this._isDesktop();
-    this.previousMobileState = this._isMobile();
-    this.previousDrawerInlineState = this._isLargeLayout();
-
-    this.state = {
-      isMastheadStacked: this._isMobile(),
-      isNavOpen: this._isDesktop(),
-      isDrawerInline: this._isLargeLayout(),
-    };
-  }
+const App_ = (props) => {
```

Comment:
Seems like a good opportunity to rename?

```suggestion
const App = (props) => {
```

Reply:
👍 on renaming